certbot/CHANGELOG.md

```diff
@@ -3,6 +3,14 @@
 Certbot adheres to [Semantic Versioning](https://semver.org/).
 
 <!-- towncrier release notes start -->
+## Upcoming
+
+### Changed
+
+ - `achallenges.KeyAuthorizationAnnotatedChallenge`, `achallenges.DNS`, and `achallenges.Other` have a
+    new field `identifier`, of type `acme.messages.Identifier`. This should be used in place of the
+    `domain` field, which is now deprecated. Objects created with the `domain` field will have that
+    value propagated into `identifier`, and vice versa. Setting both at the same time errors.
```

Comment:
ooo! we changed the way we modify the changelog since you last touched it (due to us repeatedly hitting merge conflicts in the file)

in short, you should revert all changes to this file and move the text of bullet point here to a file like `newsfragments/10491.changed`. for details, see https://github.com/certbot/certbot/blob/259dfadb43cc52cb06ba57883766497faaf72b59/.github/pull_request_template.md?plain=1#L4-L5